src/main/kotlin/no/nav/cv/eures/konverterer/CandidatePersonConverter.kt

```diff
@@ -1,25 +1,37 @@
 package no.nav.cv.eures.konverterer
 
 import no.nav.arbeid.cv.avro.Cv
+import no.nav.arbeid.cv.avro.Spraakferdighet
+import no.nav.cv.eures.konverterer.language.LanguageConverter
 import no.nav.cv.eures.model.*
 import no.nav.cv.eures.samtykke.Samtykke
 
-class CandidatePersonConverter (
+class CandidatePersonConverter(
         private val cv: Cv,
         private val samtykke: Samtykke
 ) {
 
-    fun toXmlRepresentation()
-            = CandidatePerson(
+    fun toXmlRepresentation() = when (samtykke.personalia) {
+        false -> throw Exception("Personalia is mandatory at this stage") // TODO : Consider how to handle when people don't want to share personalia
```

Comment:
Kunne vi kanskje håndtert dette i frontend? Satt for eksempel personalia til en valgt disabled checkbox med en hover som sier at det er et krav for å kunne dele CVen til Eures?